src/coreclr/vm/ceeload.h

```diff
@@ -338,7 +338,12 @@ struct VASigCookie
     unsigned        sizeOfArgs;             // size of argument list
     Volatile<PCODE> pNDirectILStub;         // will be use if target is NDirect (tag == 0)
     PTR_Module      pModule;
+    PTR_Module      pLoaderModule;
     Signature       signature;
+    DWORD           classInstCount;
+    TypeHandle*     classInst;
+    DWORD           methodInstCount;
+    TypeHandle*     methodInst;
```

Comment:
Instead of this can we create a `SigTypeContext`? Or perhaps two `Instantiation` fields?